src/commandEventsView/CommandEventsViewProvider.js

```diff
@@ -16,48 +17,61 @@ export default class CommandEventsViewProvider {
     }
 
     view(domainObject, objectPath) {
-        let table = new CommandEventsTable(domainObject, openmct);
         let component;
-        let markingProp = {
+        let _destroy = null;
+
+        const table = new CommandEventsTable(domainObject, openmct, this.options);
+        const markingProp = {
             enable: true,
             useAlternateControlBar: false,
             rowName: '',
             rowNamePlural: ''
         };
+
         const view = {
```

Comment:
could we name this `currentView` to avoid confusion since it's in the `view` method?